src/data-table/filter-menu.js

```diff
@@ -127,6 +146,7 @@ function Options(props: OptionsPropsT) {
             // handled on the wrapper element
             // eslint-disable-next-line jsx-a11y/click-events-have-key-events
             <li
+              id={`bui-${index}`}
```

Comment:
Maybe not that important, but probably will want to use the `getBuiId` utility because there could be id collisions